R/pool-package.R

```diff
@@ -8,3 +8,8 @@
 #' @importFrom later later
 #' @import rlang
 NULL
+
+## usethis namespace: start
+#' @importFrom utils packageVersion
+## usethis namespace: end
```

Comment:
Might be worth cleaning up the imports in a follow up commit, just for future you

Reply:
You mean the other imports?

Reply:
yea just moving them to the usethis managed section is all i meant